src/execution/vm/module.cpp

```diff
@@ -0,0 +1,295 @@
+#include "execution/vm/module.h"
+
+#include <memory>
+#include <mutex>  // NOLINT
+#include <string>
+#include <utility>
+
+#include <tbb/task.h>  // NOLINT
+
+#define XBYAK_NO_OP_NAMES
+#include "xbyak/xbyak/xbyak.h"
```

Comment:
Specifically, we should add `third_party/xbyak/xbyak` to our include directories so that we only have to write `#include "xbyak.h"`

Reply:
We should stick to how the other third-party libs are included so that we have `#include "<project>/<subdirs>/<header>"` (e.g., `#include "tbb/tbb.h"` or `#include "llvm/ADT/Blah.h"`).

Taken from the TPL repo, in the cmake file add `include_directories(SYSTEM "${THIRD_PARTY_DIR}/xbyak")`